Tests/Unit/LoRaTools/RegionCN470RP2TestData.cs

```diff
@@ -63,13 +62,14 @@ public static IEnumerable<object[]> TestRegionFrequencyData()
            };
 
         public static IEnumerable<object[]> TestRegionLimitData =>
-           new List<object[]>
-           {
-               new object[] { region, 470, 8, 0 },
-               new object[] { region, 510, 10, 0 },
-               new object[] { region, 509.8, 100, 0 },
-               new object[] { region, 469.9, 110, 0 },
-           };
+            from x in new (Hertz Frequency, ushort DataRate)[]
+            {
+                (Mega(470.0),   8),
+                (Mega(510.9),  10),
```

Comment:
Here the test data is changed from what it was before (510).

Reply:
Wow, you have eagle eyes! Thanks for spotting!

[![Eagle eyes!](https://i.giphy.com/media/dYdGA39uJWMs0EJdy8/giphy.webp)](https://giphy.com/gifs/bbcearth-animal-dYdGA39uJWMs0EJdy8)

Reply:
Fixed with 5bcfaaba0f0cd1b5f6b4e5fad22796b0ae01fcd5.